### PR TITLE
Add Gaojin to contributors.yml.

### PR DESCRIPTION
**Pull request by invenfantasy**
Add Gaojin to contributors.yml.

### CODE REVIEW

**Comment by invenfantasy** [comment]
cc/ @gyliu513 @haosdent, mesos repo doesn't have `.travis.yml` file, so the build has failed.

**Comment by haosdent** [comment]
You can ignore the failure because Mesos didn't use travis

File: docs/contributors.yaml [hunk 1]
```diff
@@ -511,3 +511,11 @@
     - s.dhilipkumar@gmail.com
   jira_user: dhilipkumars
   reviewboard_user: dhilipkumars
+
+- name: Gaojin CAO
```
Comment (gyliu513) [inline docs/contributors.yaml]: Can you please put your name right under Line 134 https://github.com/apache/mesos/blob/master/docs/contributors.yaml#L134 to keep alpha order for this file?

**Comment by gyliu513** [comment]
cc @vinodkone , can you please add @invenfantasy to Mesos contributor list? Thanks.

File: docs/contributors.yaml (hunk 1 above)
Comment (invenfantasy) [inline docs/contributors.yaml]: OK, and it will be my pleasure to re-order other contributors, can i?

Comment (gyliu513) [inline docs/contributors.yaml]: Sure, please go ahead ;-)

**Comment by invenfantasy** [comment]
close this, move to ref #155

**Pull request closed without merging**